rtmp-client/src/main/cpp/flvmuxer/xiecc_rtmp.c

```diff
@@ -349,7 +349,7 @@ int rtmp_sender_write_audio_frame(uint8_t *data,
         val = RTMP_Write(rtmp, output, output_len);
         free(output);
     }
-    return (val > 0) ? 0: -1;
+    return (val > 0) ? RTMP_SUCCESS: RTMP_ERROR_GENERIC;
```

Comment:
In the error case, why not return the error code returned by `RTMP_Write`?

Reply:
Since the caller function `Java_net_butterflytv_rtmp_1client_RTMPMuxer_writeAudio` is not being used anywhere, I was not sure what to return, I replaced -1 with a generic error.

I will change this to return `RTMPResult` and return `val`.